### PR TITLE
add automation tests

### PR DESCRIPTION
**Pull request by ksimon1**
This PR adds basic functional test, which deploys minishift, builds image for kubevirt-node-labeller and tries to deploy it into cluster. Then it checks if minishift node has cpu model, cpu feature labels. Due to travis restriction, there is no access to kvm device, so libvirt has to use --virttype qemu. This test tests only if labeller updates node with new labels (it doesn't check if e.g. cpu models are really supported by host or config map for cpu plugin works correctly...)
This PR add ansible for generating release manifests.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+RUN go get -u github.com/golang/dep/cmd/dep && dep ensure
+
+RUN go test ./...
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o /node-labeller cmd/node-labeller/node-labeller.go
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
-ENV container docker
+RUN mkdir -p /plugin/dest
```
Comment (fromanirh) [inline Dockerfile]: do we still need this?

**Comment by ksimon1** [comment]
@MarSik, @fromanirh please can you review this?

File: ansible_vars/test_vars.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+package_version: "test"
+node_labeller_registry: "test"
+node_labeller_version: "test"
+cpu_plugin_registry: "quay.io/kubevirt"
+cpu_plugin_version: "v0.1.0"
```
Comment (MarSik) [inline ansible_vars/test_vars.yaml]: I am a bit worried about the versions here. Who will maintain them? The whole CI test can be for nothing if those are not accurate..

File: deploy/v0.0.5/kubevirt-node-labeller-ds.yaml [hunk 3]
```diff
@@ -17,12 +17,11 @@ spec:
       serviceAccount: kubevirt-node-labeller
       containers:
       - name: kubevirt-node-labeller-sleeper
-        image: quay.io/ksimon/kubevirt-node-labeller:v0.0.5
+        image: "quay.io/ksimon/kubevirt-node-labeller:v0.0.5"
```
Comment (MarSik) [inline deploy/v0.0.5/kubevirt-node-labeller-ds.yaml]: Same here, we should generate the manifests automatically during release. Maintaining this will be a nightmate otherwise. But lets do this in different PR.

File: ansible_vars/test_vars.yaml (hunk 2 above)
Comment (ksimon1) [reply ansible_vars/test_vars.yaml]: this test is only for node labeller, not for all plugins.

File: deploy/v0.0.5/kubevirt-node-labeller-ds.yaml (hunk 3 above)
Comment (ksimon1) [reply deploy/v0.0.5/kubevirt-node-labeller-ds.yaml]: this deploy manifest is automaticaly generated by: make deploy-manifest (it creates new folder with tag version + replace all tags with version from ansible_vars/vars.yaml )
